### ISSUE TITLE
Kokoro should run integration tests in examples

### ISSUE THREAD

**Issue by igorbernstein2** [googleapis/google-cloud-java#4308]
The cloud bigtable examples contain integration tests. It would be nice to setup kokoro to run them:
https://github.com/googleapis/google-cloud-java/tree/master/google-cloud-examples/src/test/java/com/google/cloud/examples/bigtable

@elisheva-qlogic can you provide the mvn command line you used to run the integration tests?

**Comment by elisheva-qlogic** [comment]
mvn -B -pl google-cloud-examples exec:java -Dexec.mainClass="com.google.cloud.examples.bigtable.ClassNameExample"

**Comment by rahulKQL** [comment]
@igorbernstein2 

Do we still want to integrate these test case with kokoro? OR should we have some other set of integration test?  As these basic tests are taking up to 3 mins on my local machine.

```sh
Running com.google.cloud.examples.bigtable.ITHelloWorld
Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 83.605 sec

Running com.google.cloud.examples.bigtable.ITTableAdminExample
Tests run: 7, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 81.218 sec

Running com.google.cloud.examples.bigtable.ITInstanceAdminExample
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 30.616 sec
```

The command to run the integration test specific to bigtable would be:
```sh
mvn clean verify \
 -Dtest="com.google.cloud.examples.bigtable/**" \
 -Dbigtable.project=[project id] \
 -Dbigtable.instance=[instance id]
```

**Comment by igorbernstein2** [comment]
These should definitely run from kokoro, but they should run in parallel similar to how the integration tests run:

https://github.com/googleapis/google-cloud-java/blob/master/google-cloud-clients/google-cloud-bigtable/pom.xml#L104-L127

**Comment by rahulKQL** [comment]
@igorbernstein2 This issue can be closed, as with the #5215 @billyjacobson moved bigtable examples into Java-docs-samples repository.
